### PR TITLE
Create Password protected archive

### PR DESCRIPTION
**Pull request by dos-ise**
Create Password protected archive

### CODE REVIEW

File: src/SharpCompress/Common/Zip/PkwareTraditionalCryptoStream.cs [hunk 1]
```diff
@@ -102,7 +102,9 @@ protected override void Dispose(bool disposing)
             }
             isDisposed = true;
             base.Dispose(disposing);
-            stream.Dispose();
```
Comment (adamhathcock) [inline src/SharpCompress/Common/Zip/PkwareTraditionalCryptoStream.cs]: Why comment this out?

**Comment by dos-ise** [comment]
When it commented in I have gotten an objectdisposed exception when trying to write the stream

**Comment by adamhathcock** [comment]
This was easier than I thought.  I'll probably make a fresh branch and use what you did and figure out the disposal issue too.

Thanks for the code!

**Comment by dos-ise** [comment]
Did you have time to look into my pull request?

**Comment by adamhathcock** [comment]
I've started working on it here: https://github.com/adamhathcock/sharpcompress/compare/master...zip_encryption

There are still issue around fixing the headers of a zip when creating.

**Comment by dos-ise** [comment]
Thanks for your great work. Did you see that i have added another feature in my pull request?
A feature to add empty folders.

By the way ist there alreay a way to read archivecomments? I only found a way to set them..

**Comment by adamhathcock** [comment]
Please make different branches and pull requests for fixes/features.  I don't get notified of new commits in existing pull requests.

**Comment by Genae** [comment]
Really looking forward to getting this feature, thanks!

**Comment by adamhathcock** [comment]
I've been working on this in a branch, it wasn't as straight forward as I thought to get it implemented correctly.  I may pick it up again soon if I get time (which is unlikely)

https://github.com/adamhathcock/sharpcompress/tree/zip_encryption

**Pull request closed without merging**